migrations/versions/0123_add_noti_to_email_reply.py

```diff
@@ -0,0 +1,31 @@
+"""
+
+Revision ID: 0123_add_noti_to_email_reply
+Revises: 0122_add_service_letter_contact
+Create Date: 2017-09-27 09:42:39.412731
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+revision = '0123_add_noti_to_email_reply'
+down_revision = '0122_add_service_letter_contact'
+
+
+def upgrade():
+    op.create_table('notification_to_email_reply_to',
+    sa.Column('notification_id', postgresql.UUID(as_uuid=True), nullable=False),
+    sa.Column('service_email_reply_to_id', postgresql.UUID(as_uuid=True), nullable=False),
+    sa.ForeignKeyConstraint(['notification_id'], ['notifications.id'], ),
+    sa.ForeignKeyConstraint(['service_email_reply_to_id'], ['service_email_reply_to.id'], ),
+    sa.PrimaryKeyConstraint('notification_id', 'service_email_reply_to_id')
+    )
+    op.create_index(op.f('ix_notification_to_email_sender_notification_id'), 'notification_to_email_reply_to', ['notification_id'], unique=False)
```

Comment:
We expect there to be only one notification_id in the mapping table. We could make the index unique.